src/content/app/entity-viewer/state/bookmarks/entityViewerBookmarksSlice.ts

```diff
@@ -79,9 +81,15 @@ export const updatePreviouslyViewedEntities = createAsyncThunk(
         entityId: oldestPreviouslyViewedEntity.entity_id
       });
     }
+    const entityId = buildEnsObjectId({
+      genomeId,
+      type: 'gene',
+      objectId: gene.unversioned_stable_id
+    });
```

Comment:
We should have entityId already available when we call the action from the component, don't we? If so, is there any reason to rebuild the entity id from its parts??

Reply:
Yes, that's right. I've updated it.